src/compiler/types.ts

```diff
@@ -892,41 +892,33 @@ module ts {
         referencedFiles: FileReference[];
 
         hasNoDefaultLib: boolean;
-        externalModuleIndicator: Node; // The first node that causes this file to be an external module
+
+        // The first node that causes this file to be an external module
+        externalModuleIndicator: Node;
         languageVersion: ScriptTarget;
         identifiers: Map<string>;
         
-        // @internal
-        nodeCount: number;
-
-        // @internal
-        identifierCount: number;
-
-        // @internal
-        symbolCount: number;
+        /* @internal */ nodeCount: number;
+        /* @internal */ identifierCount: number;
+        /* @internal */ symbolCount: number;
 
-        // @internal
         // Diagnostics reported about the "///<reference" comments in the file.
-        referenceDiagnostics: Diagnostic[];
+        /* @internal */ referenceDiagnostics: Diagnostic[];
         
-        // @internal
         // Parse errors refer specifically to things the parser could not understand at all (like 
         // missing tokens, or tokens it didn't know how to deal with).
-        parseDiagnostics: Diagnostic[];
+        /* @internal */ parseDiagnostics: Diagnostic[];

```

Comment:
At one point there was.  Specifically, we wanted to know if there had been any actual parseDiagnostics so that we could skip grammar checks.  And we didn't want referenceDiagnostics to prevent to prevent grammar checks from happening.  

However, at this point, they do seem superfluous.  So i will remove.